tests/providers/amazon/aws/operators/test_rds.py

```diff
@@ -829,6 +812,7 @@ def test_stop_db_cluster(self, mock_await_status):
         describe_result = self.hook.conn.describe_db_clusters(DBClusterIdentifier=DB_CLUSTER_NAME)
         status = describe_result["DBClusters"][0]["Status"]
         assert status == "stopped"
+        mock_await_status.assert_called()
```

Comment:
`mock_await_status` wasn't being used (but was probably intended to be) so I added it.